src/vmx/vcpu.rs

```diff
@@ -501,7 +501,12 @@ impl<H: AxVCpuHal> VmxVcpu<H> {
         VmcsGuestNW::CR3.write(0)?;
         VmcsGuestNW::DR7.write(0x400)?;
         VmcsGuestNW::RSP.write(0)?;
-        VmcsGuestNW::RIP.write(entry.as_usize())?;
+        VmcsGuestNW::RIP.write(entry.as_usize() & 0xffff)?;
+
+        VmcsGuest16::CS_SELECTOR.write(((entry.as_usize() >> 4) & 0xf000) as u16)?;
+        // On Intel requires 'base' to be 'selector * 16' in real mode.
```

Comment:
Syntax error

which base is required to be `selector * 16` in real mode.

Reply:
Fixed

Reply:
> Syntax error
> 
> which base is required to be `selector * 16` in real mode.

Fixed